src/Silakov.Homework5/Managers/ZooManager.cs

```diff
@@ -1,16 +1,17 @@
-﻿using Silakov.Homework5.Models;
+﻿using Silakov.Homework5.Interfaces;
+using Silakov.Homework5.Models;
 using System;
 using System.Collections.Generic;
 using System.Text;
 
 namespace Silakov.Homework5.Managers
 {
-    class ZooManager
+    class ZooManager : IZooManager
     {
         /// <summary>
         /// Список животных зоопарка.
         /// </summary>
-        private List<AnimalBase> Animals { get; set; }
+        public List<AnimalBase> Animals { get; set; }
```

Comment:
So what would we use, AddAnimal method or Animal property to add animals? Having 2 options seems excesive